### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_DataGrid_Keyboard: The column headers of data grid "with variable column widths" are not resizable using the keyboard with any key combination like "ctrl + shift + right arrow"

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#184]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG2.1.1;#Keyboard;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to data grid "with variable column widths" and check if it is resizable using the keyboard with any key combination like "ctrl + shift + right arrow"

### Actual:
The column headers of data grid "with variable column widths" are not resizable using the keyboard with any key combination like "ctrl + shift + right arrow"

### Expected:
The column headers of data grid "with variable column widths" should be resizable using the keyboard with any key combination like "ctrl + shift + right arrow"

### User Impact:
Keyboard only users will face difficulty in resizing the column headers width

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
